automation/src/test/scala/org/broadinstitute/dsde/test/api/SnapshotAPISpec.scala

```diff
@@ -165,84 +165,6 @@ class SnapshotAPISpec extends AnyFreeSpecLike with Matchers with BeforeAndAfterA
 
     }
 
-    "should add a BigQuery dataset for each snapshot reference added" taggedAs(Tags.AlphaTest, Tags.ExcludeInFiab) in {
-      implicit val patienceConfig: PatienceConfig = PatienceConfig(timeout = 20 seconds)
-
-      val owner = UserPool.userConfig.Owners.getUserCredential("hermione")
-      implicit val ownerAuthToken: AuthToken = owner.makeAuthToken(userLoginScopes ++ Seq("https://www.googleapis.com/auth/cloud-platform"))
-
-      withCleanBillingProject(owner) { billingProject =>
-        withWorkspace(billingProject, s"${UUID.randomUUID().toString}-snapshot references") { workspaceName =>
-
-          // we need the actual Google project for this workspace, not just its namespace
-          val googleProject = getWorkspaceGoogleProject(billingProject, workspaceName)
-
-          val drSnapshot = listDataRepoSnapshots(1, owner)(ownerAuthToken)
-          val dataRepoSnapshotId = drSnapshot.getItems.get(0).getId
-
-          val snapshotName = "snapshotReferenceAddBQDataset"
-          // add snapshot reference to the workspace. Under the covers, this creates the workspace in WSM and adds the ref
-          createSnapshotReference(billingProject, workspaceName, dataRepoSnapshotId, snapshotName)
-
-          // validate the snapshot was added correctly: list snapshots in Rawls, should return 1, which we just added.
-          val listResponse = listSnapshotReferences(billingProject, workspaceName)
-          val resources = Rawls.parseResponseAs[SnapshotListResponse](listResponse).gcpDataRepoSnapshots
-          resources should have size 1
-
-          // check that the bq dataset has been created
-          val workspaceId = resources.head.getMetadata.getWorkspaceId
-          val dataset = getDataset(generateReferenceName(workspaceId), googleProject.value, ownerAuthToken)
```

Comment:
I believe the `generateReferenceName` method is now unused and can be deleted

Reply:
Yup, it is unused, fixed!